xep-0068.xml

```diff
@@ -23,6 +23,12 @@
   <registry/>
   &hildjj;
   &stpeter;
+  <revision>
+    <version>1.2.1</version>
```

Comment:
This is fine if it's editorial changes, otherwise this needs to be 1.3.0. I'll let you ack/nack this :)